File: Programmers/Level4/bestSet(pjct)/BestSet.java
import java.util.Arrays;

public class BestSet {

    public int[] bestSet(int n, int s){
        int[] answer = {-1};
        if(n>s)
        {
        	return answer;
        }
        answer = new int[n];
        for(int i=0; i<n; ++i)
        {
        	if(i<n-s%n)
        	{
        		answer[i] = s/n;
        	}
        	else
        	{
        		answer[i] = s/n+1;
        	}
        }

        return answer;
    }
    public static void main(String[] args) {
        BestSet c = new BestSet();
        System.out.println(Arrays.toString(c.bestSet(3,13)));
    }

}

File: Programmers/Level4/bestSet(pjct)/bestSet.py
def bestSet(n, s):
    if n > s:
        return [-1]
    return [s//n if i<n-s%n else s//n+1 for i in range(n)]

# 아래는 테스트로 출력해 보기 위한 코드입니다.
print(bestSet(3,13))